packages/components/src/time-since/index.tsx

```diff
@@ -88,6 +88,13 @@ function formatDate( date: Date, format: string, locale?: string ): string {
 		// Medium date format without time
 		formatOptions.dateStyle = 'medium';
 		delete formatOptions.timeStyle;
+	} else if ( format === 'lll' ) {
+		// Always d MMM y, e.g., 12 Jun 2024
+		return new Intl.DateTimeFormat( locale, {
+			day: 'numeric',
+			month: 'short',
+			year: 'numeric',
+		} ).format( date );
```

Comment:
In the `else if ( format === 'llll' ) {` below, we're modifying `formatOptions` directly and letting it fall through to the final `return` statement. Should we be consistent with how these conditions apply their behaviors?